pandora-client-web/src/config/definition.ts

```diff
@@ -3,6 +3,7 @@ import { z } from 'zod';
 export const WEBPACK_BASE_CONFIG = {
 	DIRECTORY_ADDRESS: z.string().default('http://127.0.0.1:25560'),
 	EDITOR_ASSETS_ADDRESS: z.string().default('http://127.0.0.1:26969/assets'),
+	EXTRA_ASSETS_ADDRESS: z.string().default(''),
```

Comment:
What exactly is the plan for this, also where the preview images, in assets repo?

Reply:
/me explained the idea to Sekkmer:
Sekkmer: it could be more controlled though
Sekkmer: like what if an image is missing
Sekkmer: what if it fails to load

Reply:
@Sekkmer:
It was created with following points in mind:
- We don't want fully rendered scenes in any public repo, to adhere to TOS
- The images need to be served before login - so before selecting character too, meaning they can't be part of asset handling pipeline
- There shouldn't be any error produced when running local development client

This way of handling it felt as the most straightforward one:
It allows specifying URL where to load things from (can be the same as public assets url, but doesn't have to be - it is dependent entirely on how we do deployment; right now I just updated the NGinx setup to host it on a specific path).
I believe we don't need some handling for when this fails to load - it would behave the same as any other image on the internet that fails to load. It would also retry by itself the next time the image is focused while the login preview is cycling around.